tests/util/test_object.py

```diff
@@ -1,7 +1,49 @@
 ''' Tests for datamegh.util.object util. '''
 
 import pytest
-from datamegh.util.object import linearize, delinearize, merge
+from datamegh.util.object import linearize, delinearize, merge, dict_to_list
+
+
+def test_dict_to_list_1():
```

Comment:
Not sure if naming tests with post fix `_number` is a good idea.

Reply:
Yes, you can make the function name a bit verbose - which will do the job for eg: `test_dict_to_list_converts_dict_to_list`